CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 #### Features
 
+* [#1463](https://github.com/ruby-grape/grape/pull/1463): Fix array indicies in error messages - [@ffloyd](https://github.com/ffloyd).
```

Comment:
Lol English is my third language too :)

I meant it's in the wrong section, there's features and fixes. This should be in fixes.